profiles/lib/destination.lua

```diff
@@ -18,11 +18,18 @@ end
 function Destination.get_destination(way, is_forward)
   ref  = Destination.get_directional_tag(way, is_forward, 'destination:ref')
   dest = Destination.get_directional_tag(way, is_forward, 'destination')
+  street = Destination.get_directional_tag(way, is_forward, 'destination:street')
   if ref and dest then
     return ref .. ': ' .. dest
+  elseif ref then
+    return ref
+  elseif dest then
+    return dest
+  elseif street then
+    return street
```

Comment:
capturing from desk: this could be revisited to use the former syntax `return ref or dest or street or ' '`, as long as we do the check for street anyhow.